src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    white,
+    black,
+    yellow,
+    red,
+    blue
```

Comment:
Emun elements should be written in caps
```suggestion
    WHITE,
    BLACK,
    ...
```